### ISSUE TITLE
warm_start and other functions

### ISSUE THREAD

**Issue by bharswami** [coin-or/filterSD#2]
I am building the library for filtersd and calling it from a C file. I am getting errors since subroutines like warm_start aren’t defined anywhere. Could you please tell me where these subroutines are defined?

**Comment by jacobwilliams** [comment]
FYI: https://www.coin-or.org/download/source/filterSD/

The newest zip has files that aren't in this repo (see `shared.f` which has warm_start). I'm not sure why that is.

**Comment by tkralphs** [comment]
The releases are made from the stable branches. The release 2.0.1, which matches the latest zip file [here](https://www.coin-or.org/download/source/filterSD/) (these zip files are archives from the days when COIN-OR was managed in subversion), is the tag in this repository named [`releases/2.0.1`](https://github.com/coin-or/filterSD/releases/tag/releases%2F2.0.1), which is a snapshot of the branch `stable/2.0`. The `master` branch here is the development branch, which doesn't have the file `shared.f` for some reason. Maybe @frankecurtis knows why? It looks like it's been that way since the beginning. Since there isn't any development happening, we could probably set the default branch to be `stable/2.0`.

**Comment by frankecurtis** [comment]
I'm not sure why the master branch doesn't have the file shared.f.  I can only guess that it happened in the migration to GitHub somehow.  In any case, for the original question about warm_start, it is included in shared.f, so I think @tkralphs is right that we should make the default branch to be stable/2.0

**Comment by svigerske** [comment]
Branches `master` and `stable/2.0` have completely different commit histories (they do not share a common beginning):
- https://github.com/coin-or/filterSD/commits/master/
- https://github.com/coin-or/filterSD/commits/stable/2.0/

Both have a "Release 2.0" commit, but they are different. Both have the same bugfix commit for `schurQR.f`. Only master has the commits for additional coin-or meta files (coin-or project specification) and the content from the trac-wiki committed to README.

(stable/1.0 is yet another commit history)

**Comment by tkralphs** [comment]
It looks like `master` was created first, then `stable/1.0` by @frankecurtis a few months later, but just by directly committing files, not by branching. I think this code was previously maintained outside of source control and this was probably trying to replicate existing history. The file `shared.f` was added in the release 2.0 commit in `stable/2.0`, but I guess it was an oversight not to also add it in the corresponding commit in the `master` branch. 

In any case, all the actual release tags are in the stable branch(es). I guess we can just make `stable/2.0` the default branch.

**Comment by tkralphs** [comment]
OK, that is done now.

**Comment by frankecurtis** [comment]
> It looks like `master` was created first, then `stable/1.0` by [@frankecurtis](https://github.com/frankecurtis) a few months later, but just by directly committing files, not by branching. I think this code was previously maintained outside of source control and this was probably trying to replicate existing history. The file `shared.f` was added in the release 2.0 commit in `stable/2.0`, but I guess it was an oversight not to also add it in the corresponding commit in the `master` branch.
> 
> In any case, all the actual release tags are in the stable branch(es). I guess we can just make `stable/2.0` the default branch.

Right.  The code was initially shared by Roger Fletcher in 2011 as a zip.  I set that up in the old COIN repo as version 1.0.  A couple of years later, in 2013, he sent another zip with all of the code that he wanted posted as version 2.0.  I posted that, which didn't end up leading to a commit history with version 1.0.  A couple of years later, in 2015, there was one bug fixed, which I suppose I might have fixed in both versions.  Roger passed away in 2016 and there has been no active development since then.  The code migrated to GitHub in 2020, it seems.

Good to change the default branch, as @tkralphs handled already.  Just posting this for the sake of posterity.
